core/ide/src/main/java/org/alice/stageide/sceneeditor/StorytellingSceneEditor.java

```diff
@@ -595,8 +596,9 @@ private void handleManipulatorSelection(SelectionEvent e) {
         } else {
           this.setSelectedField(field.getDeclaringType(), field);
         }
```

Comment:
this function is starting to get scary, but it is also creeping into drag adaptor territory, so i think i'm backing away quietly to not wake the beast.

Reply:
So many dispatcher methods like this.
More "Uh oh" than "OO", amirite?

I join you in slowly stepping away.